source/query/filter.txt

```diff
@@ -108,6 +108,172 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
+         }
+      ]
+
+.. note::
+
+   If you do not have a MonogDB deployment or if you would like to 
+   query a complete sample data set, see
+   :atlas:`Sample Data for Atlas Clusters</sample-data/>` for 
+   instructions on creating a free-tier cluster with sample data.
+
+Filter for a Match
+~~~~~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where 
+   ``name`` is "Andrea Le" using the 
+   :manual:`$eq operator </reference/operator/query/eq/>`.
+
+   .. code-block:: shell
+
+      { name: { $eq: "Andrea Le" } }
+
+   If you query the provided sample documents, the following document 
+   is returned because it's ``name`` matches "Andrea Le" exactly.
+
+   .. code-block:: JSON
+      :copyable: false
+
+      {
+         "_id":ObjectId("5a9427648b0beebeb69579cc"),
+         "name":"Andrea Le",
+         "email":"andrea_le@fakegmail.com",
+         "age":34,
+         "scores":[85, 95, 75],
+         "dateCreated":2012-03-26T23:20:16.000+00:00
+      }
+
+Filter by Comparison
+~~~~~~~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where
+   ``age`` is greater than or equal to ``40`` using the 
+   :manual:`$gte operator </reference/operator/query/gte/>`.
+
+   .. code-block:: shell
+
+      { age: { $gte: 40 } }
+
+   If you query the provided sample documents, the following document 
+   is returned because it's ``age`` is greater than ``40``.
+
+   .. code-block:: JSON
+      :copyable: false
+
+      {
+         "_id":ObjectId("5a9427648b0beebeb69579cf"),
+         "name":"Greg Powell",
+         "email":"greg_powell@fakegmail.com",
+         "age":52,
+         "scores":[65, 75, 80],
+         "dateCreated":1987-02-10T00:29:36.000+00:00
+      }
+
+Filter by Date
+~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where 
+   the ``dateCreated`` is later than June 22nd, 2000 at 12:30 PM using 
+   the :manual:`$gt operator </reference/operator/query/gt/>` and 
+   :manual:`Date() method </reference/method/Date/>`.
+
+   .. code-block:: shell
+
+      { dateCreated: { $gt: Date('2000-06-22T12:30:00.000+00:00') } }
+
+   If you query the provided sample documents, the following document 
+   is returned because it's ``dateCreated`` is later than June 22, 2000 
+   at 12:30 PM.
+
+   .. code-block:: JSON
+      :copyable: false
+
+      {
+         "_id":ObjectId("5a9427648b0beebeb69579cc"),
+         "name":"Andrea Le",
+         "email":"andrea_le@fakegmail.com",
+         "age":34,
+         "scores":[85, 95, 75],
+         "dateCreated":2012-03-26T23:20:16.000+00:00
+      }
+
+Filter by Elements in an Array
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where ``scores`` 
+   contains at least one number greater than ``80`` and less than 
+   ``90`` using the
+   :manual:`$elemMatch operator </reference/operator/query/elemMatch/>`.
```

Comment:
Consider rephrasing to something similar to the following:

The following $filter query uses the :manual:`$elemMatch operator </reference/operator/query/elemMatch/>` to find documents where at least one value in the ``scores`` array is greater than ``80`` and less than 90.

Reply:
Rephrasing all examples this way